db/migrate/20220316205242_change_profile_rules_to_profile_links.rb

```diff
@@ -0,0 +1,17 @@
+class ChangeProfileRulesToProfileLinks < ActiveRecord::Migration[7.0]
+  class ProfileLink < ActiveRecord::Base
+  end
+
+  def change
```

Comment:
Instead of a change, I'd define a separate `up` and `down` method and implement the migration in both directions separately. This way you won't need the `reversible` block, also note that you need to move the values from the `entity_id` to the `rule_id` back in the `down` migration to have it correctly.